src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +34,14 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        IntStream.range(0, numbers.size())
+                .filter(i -> i % 2 != 0)
+                .forEach(i -> numbers.set(i, numbers.get(i) - 1));
+        return numbers.stream()
```

Comment:
Можна це зробити одним стрімом :
```suggestion
        return IntStream.range(0, numbers.size())
                           .map(i -> i % 2 != 0 ? (numbers.get(i) - 1) : numbers.get(i))
                           ....
```